nautobot/utilities/query_functions.py

```diff
@@ -38,7 +38,7 @@ class JSONBAgg(Aggregate):
 
     function = None
     output_field = JSONField()
-    # TODO: Django's JSONBAgg has `allow_distinct=True`, we might want to think about adding that at some point?
+    # TODO(Glenn): Django's JSONBAgg has `allow_distinct=True`, we might want to think about adding that at some point?
```

Comment:
What clarification is needed here?

Reply:
I have no idea.

Reply:
Do you suggest we just go ahead and add it? Seems like it wouldn't hurt us to allow it.

Reply:
I'm not sure of the implications. That's why it's a TODO :) Just wasn't sure why I needed to be tagged on this one to provide clarification, since it seems like the comment is pretty self-explanatory.